### PR TITLE
[Merge v3.0.0] into Master

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Release Overview
This release has a minor improvement in the UD implementation and a major improvement with adding Phase 1/v1 of the Litecoin Card integration
- Improve Unstoppable Domains business logic in the field (empty field, clear domain resolution, etc.)
- Implementation of Litecoin Card for with Login and Registration

### Unstoppable Domain improvements
- Disable the Button prior to filling the search domain
- Clears the address field when the `Lookup` button is pressed
- The successful domain resolution presents toast

### Litecoin Card v1: Login, Registration
- Hybrid UIKit / SwiftUI integrations
- Legacy mockups used as a guide but will be polished further 
- Using the Partner Library API

### Video Clips
|UD Minor Update | Litecoin Card v1 |
|------------------|-----------------|
|uddisableenable|CardV2|

### CODE REVIEW

File: loafwallet/CardView.swift [hunk 1]
```diff
@@ -69,14 +72,20 @@ struct CardView: View {
                 //MARK: - Login Textfields
                 Group {
```
Comment (kcw-grunt) [inline loafwallet/CardView.swift:81]: Fixes this issue: @losh11

Comment (kcw-grunt) [reply loafwallet/CardView.swift:81]: `The cards login now gives a good login failed alert. However I think the login button should be disabled until the text in the email input is validated as email format.`

**Review by mosadialiou — APPROVED** [review]
LGTM

**Comment by kcw-grunt** [comment]
Quickest approval from Apple known to man..

Activity | User | Date
-- | -- | --
Ready for Sale | Apple | Jan 9, 2021 at 11:20 PM
In Review | Apple | Jan 9, 2021 at 10:49 PM
Waiting for Review | mrkerrywashington@icloud.com | Jan 9, 2021 at 10:46 PM

**Pull request merged**